Gemfile.lock

```diff
@@ -11,7 +11,7 @@ GEM
     eventmachine (1.2.7)
     ffi (1.16.3)
     forwardable-extended (2.6.0)
-    google-protobuf (3.25.2)
+    google-protobuf (3.25.3)
```

Comment:
This was part of my troubleshooting to resolve issues with `bundle install` on Ruby 3.3. It didn't help and I had to resort to downgrading to Ruby 3.2 (8d3e09418a62c792cbfbe11bc351af40228bab98), but figured to leave the update anyways.